Documentation/Signoffs/Orientation-Signoff.md

```diff
@@ -0,0 +1,116 @@
+
+The module has 4 modes. The first mode is for connecting the sensor set to a single host using I2C. This is the mode that the base robot will be utilizing. The other three modes are for connecting the sensor set to both
+the host and additional modules for dual communication. These modes will be documented in the robot's user manual so that future teams know the functionality possibilities of the senor module. The accelerometer 
+and gyroscope provide the chip with 6 degrees of freedom to measure the orientation of the robot. ST gives a model of the measurement capabilities:
+
+<p align = "center">
+<img src = "https://github.com/lchapman42/Control-Sensing-Wireless-Charging-Robot/blob/main/Documentation/Images/Orientation/Sensing_Axis.png?raw=true"alt="Axis"/>
+</p>
+
+The module is capable of capturing the acceleration in the x,y, and z planes (top image). In addition to measuring the acceleration, it can also produce angle values shown in the bottom image. ST claims that the accelerometer is capable of measuring $\pm2\pm4\pm8\pm16$ g from 1.6 Hz to 6.7 kHz update rate. The gyroscope is capable of $\pm125\pm250\pm500\pm1000\pm2000$ degrees per second from 12.5 Hz to 6.7 kHz update rate. The update rate of 2 kHz will provide the master control with ample readings. At maximum speed, the sensor will read new data every 0.012 inches: 
+
+
+$2 \frac{ft}{s}\cdot \frac{1}{2000s} = 0.001 ft \text{ or }0.012in$
+
+The sensor set also allows for tilt detection, so when the robot is climbing up an incline and needs to increase power to the motors, the tilt detection can interrupt the routine to send the master control the angular data. 
+
+### LIS3MDL Magnetometer (Compass) 
+The LIS3MDL Magnetometer will draw a maximum of 270 $\mu A$ in high resolution mode. In low power mode, the sensor only draws 40 $\mu A$. The supply voltage is rated from 1.9 V to 3.6 V. The 3.3 V power supply will be able to supply a voltage within the acceptable range. The magnetometer is capable of measuring full scales of $\pm 4/ \pm 8/ \pm 12/ \pm 16$ gauss. The datasheet suggests that currents higher than 10 $mA$ cannot be near the sensor. The magnetic field generated by the current will contribute to the measured Earth field which will cause false readings. The suggested distance is a "few millimeters". The tray that holds the device will also help isolate the sensor set from the current-generated field. 
+
+The device can be tested using the "self test". The self test tests the sensor without having to have any external force applied to the module. 
+
+<p align = "center">
+<img src = "https://github.com/lchapman42/Control-Sensing-Wireless-Charging-Robot/blob/main/Documentation/Images/Orientation/Compass_Directions.png?raw=true"alt="Compass Direction"/>
+</p>
+
+The sensor is capable of measuring the magnetic fields in the x, y, and z directions. The reference to Earth will help calculate which direction the robot is facing. 
```

Comment:
How did you do the self test?

Reply:
The self test cannot be done without the sensor, so I haven't done it yet. That was more for when we go to implement it in the robot for Capstone 2.